commands/show-long-connections.go

```diff
@@ -74,7 +102,7 @@ func showConns(connResults []uconn.LongConnAnalysisView) error {
 			result.Src,
 			result.Dst,
 			strings.Join(result.Tuples, " "),
-			f(result.MaxDuration),
+			duration(time.Duration(int(result.MaxDuration*float64(time.Second)))),
```

Comment:
I'm not sure we want this in the csv output. It would make sorting or filtering difficult both on the command line or in a spreadsheet program.

Reply:
Don’t most people use the regular non human readable output for parsing?

Reply:
I would think so. I'm not sure what you're getting at, sorry.

Reply:
Yep, you’re right, I missed that the csv output was being edited too

Reply:
Yep, you’re right, I missed that the csv output was being edited too